targets/targets.json

```diff
@@ -2123,7 +2123,7 @@
             "NXP_LPADC",
             "MBED_TICKLESS"
         ],
-        "components_add": ["FLASHIAP"],
+        "components_add": ["FLASHIAP", "QCA400X_WIFI"],
```

Comment:
I am confused... you dropped the `COMPONENT_` part and left the `_WIFI` in... Does this still work fine? 
Is there any reason to keep the `_WIFI` suffix? @0xc0170 can you give your recommendations on the naming or do you know who can?

Reply:
COMPONENT is not needed as it will be added during build as it is part of the components_add section. I left the WIFI in the folder name as this is the convention followed by cellular.

Reply:
It should be fine as it is now